### PR TITLE
feat: about us section

### PR DESCRIPTION
**Pull request by IamDikshaR**

### CODE REVIEW

File: src/atoms-diksha/headings/six/color-primary/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/headings/six/color-primary/index.astro:1]: Update path: src/atoms-diksha/headings/six/primary/index.astro

File: src/atoms-diksha/headings/two/6xl/black/color-primary/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/headings/two/6xl/black/color-primary/index.astro:1]: 1. Remove color- from all atoms.

File: src/components/home/about/index.astro
Comment (tanishdotdev) [inline src/components/home/about/index.astro:1]: 1. Download images in public and then use.
2. Use picture atom inplace of images.
3. Rename atoms like HeadingTwoBlackPrimary
4. Rename atom HeadingSixPrimary.
5. Create an object for data and then access it.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: src/atoms-diksha/headings/two/6xl/black/color-primary/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/headings/two/6xl/black/color-primary/index.astro:1]: done

File: src/atoms-diksha/headings/six/color-primary/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/headings/six/color-primary/index.astro:1]: done

File: src/components/home/about/index.astro
Comment (IamDikshaR) [reply src/components/home/about/index.astro:1]: done

**Pull request closed without merging**